addon/styles/addon.scss

```diff
@@ -5,26 +5,63 @@
   flex-direction: column;
   background-color: $frost-color-lgrey-3;
 
-
   > .view-tabs {
-    display: flex;
-    flex-direction: row;
-    height: 20px;
-    margin: 10px 0 0 10px;
+    margin: 10px 0 20px 30px;
+
+    .tabs {
+      font-size: 0;
+      display: flex;
+      flex-direction: row;
 
-    .frost-link.frost-view-detail {
-      margin-right: 10px;
-      color: $frost-color-grey-4;
+      > .frost-link.frost-view-detail {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        position: relative;
+        text-align: center;
+        height: 37px;
+        padding: 0 20px 0 20px;
+        font-size: $frost-font-m;
+        color: $frost-color-grey-5;
+        margin-left: -1px;
+
+        &:nth-child(n + 2) {
+          .tab-text {
+            &::after {
+              content: '';
+              position: absolute;
+              top: 12px;
+              left: 0;
+              width: 1px;
+              height: 15px;
+              background-color: $frost-color-lgrey-2;
+            }
+          }
+        }
+
+        &.active {
+          color: $frost-color-grey-2;
+        }
+
+        &:hover {
+          background-color: rgba($frost-color-blue-1, 0.05);
+          .tab-text {
+            color: $frost-color-grey-3;
+            border-bottom: 1px solid $frost-color-grey-3;
+          }
+        }
 
-      &:nth-child(n + 2) {
-        border-left: 1px solid $frost-color-grey-4;
-        padding-left: 10px;
       }
+    }
+
+    .frost-view-follower {
```

Comment:
I'd prefer that this be scoped (e.g. `.frost-object-details-view-follower`) - I'll raise an issue